include/dali/util/half.hpp

```diff
@@ -1016,12 +1016,10 @@ namespace half_float
       assign(static_cast<float>(rhs));
     }
 
-    // Copy constructor overriding the one above
-    // It avoids half->float->half path
+    /// Trivial copy constructor.
 	HALF_CONSTEXPR
     CAFFE_UTIL_HD
-    half(const half& rhs) :
-      data_(rhs.data_) {}
+    half(const half &) = default;
```

Comment:
GCC 9 protests when this type is copied via `memcpy` - it's not trivially copyable.